derive_builder/src/doc_tpl/builder_struct.md

```diff
@@ -1 +1 @@
-Builder for `{struct_name}`.
+Builder for [`{struct_name}`](struct.{struct_name}.html).
```

Comment:
Note: Both the name and the link will break, if `{struct_name}` is re-exported, because the generated documentation will not know about the re-export. This is something we cannot fix - at least not automatically.

To see what I mean you can run this on current master
* `cd derive_builder/tests/dummy-crate`
* `cargo doc`